projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -1135,7 +1271,14 @@ public function carousel_display_comments_callback() {
 		$this->settings_checkbox( 'carousel_display_comments', esc_html__( 'Show comments area in carousel', 'jetpack' ) );
 	}
 
-	function carousel_display_exif_sanitize( $value ) {
+	/**
+	 * Calls the sanitize_1or0_option function to sanitize the passed value.
```

Comment:
I'd focus more on what than how.
```suggestion
	 * Sanitize input for the `carousel_display_exif` setting.
```